rust/src/smb/smb.rs

```diff
@@ -26,175 +26,186 @@
 // written by Victor Julien
 
 use std;
+use std::ffi::{self, CStr};
 use std::mem::transmute;
 use std::str;
-use std::ffi::{self, CStr};
 
 use std::collections::HashMap;
 
 use nom;
 
-use crate::core::*;
 use crate::applayer;
 use crate::applayer::*;
+use crate::core::*;
 use crate::filecontainer::*;
 
 use crate::smb::nbss_records::*;
 use crate::smb::smb1_records::*;
 use crate::smb::smb2_records::*;
 
-use crate::smb::smb1::*;
-use crate::smb::smb2::*;
-use crate::smb::smb3::*;
 use crate::smb::dcerpc::*;
-use crate::smb::session::*;
 use crate::smb::events::*;
 use crate::smb::files::*;
+use crate::smb::session::*;
+use crate::smb::smb1::*;
+use crate::smb::smb2::*;
 use crate::smb::smb2_ioctl::*;
+use crate::smb::smb3::*;
 
-pub const MIN_REC_SIZE: u16 = 32+4; // SMB hdr + nbss hdr
-pub const SMB_CONFIG_DEFAULT_STREAM_DEPTH:u32 = 0;
+pub const MIN_REC_SIZE: u16 = 32 + 4; // SMB hdr + nbss hdr
+pub const SMB_CONFIG_DEFAULT_STREAM_DEPTH: u32 = 0;
 
 static mut ALPROTO_SMB: AppProto = ALPROTO_UNKNOWN;
 
 pub static mut SURICATA_SMB_FILE_CONFIG: Option<&'static SuricataFileContext> = None;
 
 #[no_mangle]
-pub extern "C" fn rs_smb_init(context: &'static mut SuricataFileContext)
-{
+pub extern "C" fn rs_smb_init(context: &'static mut SuricataFileContext) {
     unsafe {
         SURICATA_SMB_FILE_CONFIG = Some(context);
     }
 }
 
-pub const SMB_NTSTATUS_SUCCESS:                    u32 = 0;
```

Comment:
The commit name is not about rustfmt.
Could we remove the formatting out of this commit ?